crates/compiler/builtins/bitcode/src/str.zig

```diff
@@ -1306,6 +1306,99 @@ test "countGraphemeClusters: emojis, ut8, and ascii characters" {
     try expectEqual(count, 10);
 }
 
+// Str.graphemes
+pub fn strGraphemes(roc_str: RocStr) callconv(.C) RocList {
+    var break_state: ?grapheme.BoundClass = null;
+    var opt_last_codepoint: ?u21 = null;
+    var index: usize = 0;
+    var last_codepoint_len: u8 = 0;
+
+    var result = RocList.allocate(@alignOf(RocStr), countGraphemeClusters(roc_str), @sizeOf(RocStr));
+    const graphemes = result.elements(RocStr) orelse return result;
+    var slice = roc_str.asSlice();
+    var iter = (unicode.Utf8View.init(slice) catch unreachable).iterator();
+
+    while (iter.nextCodepoint()) |cur_codepoint| {
+        const cur_codepoint_len = unicode.utf8CodepointSequenceLength(cur_codepoint) catch unreachable;
+        if (opt_last_codepoint) |last_codepoint| {
+            var did_break = grapheme.isGraphemeBreak(last_codepoint, cur_codepoint, &break_state);
+            if (did_break) {
+                graphemes[index] = RocStr.fromSlice(slice[0..last_codepoint_len]); //fromSlice(bs_slice);
+                slice = slice[last_codepoint_len..];
+                index += 1;
+                break_state = null;
+                last_codepoint_len = 0;
+            }
+        }
+        last_codepoint_len += cur_codepoint_len;
+        opt_last_codepoint = cur_codepoint;
+    }
+    // Append last grapheme
+    graphemes[index] = RocStr.fromSlice(slice);
+    std.debug.assert(index + 1 == result.length);
```

Comment:
I think we can remove this now that it is working

Reply:
thanks. done.